### PR TITLE
chore(docs): renaming monada to wing cloud

### LINKED ISSUE #3658: change Monada, Inc to Wing Cloud in docs, sites and licenses
### Feature Spec

replace the company name in all relevant places

### Use Cases

company name update

### Implementation Notes

_No response_

### Component

Documentation

### Community Notes

* Please vote by adding a 👍 reaction to the issue to help us prioritize.
* If you are interested to work on this issue, please leave a comment.

### PR DESCRIPTION
**Pull request by garysassano**
Closes #3658

Things that didn't get modified:
- PagerDuty link (https://monada.pagerduty.com/incidents)
- Email (ping@monada.co)
- Monada Bot and its email (monadabot@monada.co)
- Tailwind color names (monada-light, monada-dark, monada-black)
- VS Code extension link (https://marketplace.visualstudio.com/items?itemName=Monada.vscode-wing)
- npm login (--scope=@monadahq)

*By submitting this pull request, I confirm that my contribution is made under the terms of the [Wing Cloud Contribution License](https://github.com/winglang/wing/blob/main/CONTRIBUTION_LICENSE.md)*.

### CODE REVIEW

**Comment by ainvoner** [comment]
@garysassano thanks!!!!
@ShaiBer need to make sure that we have a green light for this change

**Comment by ShaiBer** [comment]
> @garysassano thanks!!!! @ShaiBer need to make sure that we have a green light for this change

Just got the approval today, so you can continue with this PR

File: apps/wing-api-checker/.projenrc.ts [hunk 1]
```diff
@@ -8,10 +8,10 @@ const project = new typescript.TypeScriptProject({
     "wing-api-check": "bin/wing-api-check",
   },
   license: "MIT",
-  authorName: "Monada",
+  authorName: "Wing Cloud",
```
Comment (MarkMcCulloh) [inline apps/wing-api-checker/.projenrc.ts:11]: Need to undo this change. The publisher is still Monada to avoid changing the id of the extension.

```suggestion
  authorName: "Monada",
```

File: CONTRIBUTORS_TERMS_OF_SERVICE.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
...
+1.1. Wing Cloud, a Delaware corporation and/or its affiliates (dba “Wing Cloud”) (“Company”) may operate release and operate certain projects under an open-source licensing regime (the “Project(s)”). If you wish to contribute to or otherwise participate in any such Project, please read carefully the following Contributors Terms of Service (“CTS”). NOTE: This document does not apply to or govern the use of (and/or contributions to) any Company’s licensed products or services which are subject to separate license agreements. For example, the Company Console Software Application, is governed by the Company Console Software License Agreement (the current versions of which could be found **[here](https://github.com/winglang/wing/blob/main/apps/wing-console/LICENSE.md)**).
```
[the remark this answers is not in the document]
Comment (eladb) [reply CONTRIBUTORS_TERMS_OF_SERVICE.md:7]: Yes, we are Wing Cloud Inc.

File: apps/wing/LICENSE.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2022-2023 Monada, Inc.
+Copyright (c) 2022-2023 Wing Cloud
```
Comment (eladb) [inline apps/wing/LICENSE.md:3]:
```suggestion
Copyright (c) 2022-2023 Wing Cloud, Inc.
```

File: docs/contributing/03-stories/story-04.md [hunk 4]
```diff
@@ -2,17 +2,17 @@
 
 > **Status**: Done
 
-This is the story of Monada Sprint 4. It is told from the perspective of Mona
```
Comment (eladb) [inline docs/contributing/03-stories/story-04.md:5]: We traditionally don't change old stories. No rewriting of history :-) Please revert.

File: apps/wing-api-checker/.projenrc.ts (hunk 1 above)
Comment (garysassano) [reply apps/wing-api-checker/.projenrc.ts]: Can keep the new `authorUrl` or need to revert that as well? Also `authorEmail` must be left unchanged?

Comment (garysassano) [reply apps/wing-api-checker/.projenrc.ts]: Also, do we need to keep`Monada` as author/publisher just for the VS Code extension or also for the `package.json` and `projenrc.ts` files?

Comment (MarkMcCulloh) [reply apps/wing-api-checker/.projenrc.ts]: In checking this, the only field that matters is "publisher", all other fields can change. The current state looks good

**Review by ShaiBer — APPROVED** [review]

**Review by ShaiBer — APPROVED** [review]

**Review by staycoolcall911 — APPROVED** [review]

File: apps/wing-api-checker/.projenrc.ts (hunk 1 above)
Comment (garysassano) [reply apps/wing-api-checker/.projenrc.ts]: @MarkMcCulloh Another thing: I noticed there were half folders with a `LICENSE` file and another half with a `LICENSE.md` file, so I renamed everything to `LICENSE.md` but then monadabot [recreated the LICENSE file for two folders](https://github.com/winglang/wing/pull/3674/commits/46171b39d43c042e9dab3da01f89c67fd4affc66). Is it possible to fix this behaviour?

Comment (garysassano) [reply apps/wing-api-checker/.projenrc.ts]: @MarkMcCulloh Another thing: I noticed there were half folders with a `LICENSE` file and another half with a `LICENSE.md` file, so I renamed everything to `LICENSE.md`, but then monadabot [recreated the LICENSE file for two folders](https://github.com/winglang/wing/pull/3674/commits/46171b39d43c042e9dab3da01f89c67fd4affc66). Is it possible to fix this behaviour?

**Comment by staycoolcall911** [comment]
@ainvoner could you please give this PR a review? Thanks!

**Review by ainvoner — APPROVED** [review]
